Dockerfile

```diff
@@ -15,7 +15,6 @@ COPY main.go main.go
 COPY api api/
 COPY config config/
 COPY controllers controllers/
-COPY internal internal/
```

Comment:
This change should go to https://github.com/openshift/sandboxed-containers-operator/pull/417/commits/e01db87b68081a42f4484c360ebbef9fb515822c which is actually removing the directory. It is important to preserve `git bisect` when we can.

Reply:
Oh, I didn't realize that